persistent/persistent.cabal

```diff
@@ -96,6 +99,7 @@ library
     Database.Persist.TH
     Database.Persist.TH.Internal
     Database.Persist.Types
+    Database.Persist.Types.Span
```

Comment:
Can we put doc comments in this module? I think mostly just having `-- | @since 2.16.0.0` in everything exported here would be great

(and also maybe yeah uh renaming to `SourceSpan` or something that has fewer clashes)